product_selled_delivered_weekly/views/product_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - David Vidal
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="product_template_tree_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_tree_view" />
+        <field name="groups_id" eval="[(4, ref('base.group_user'))]" />
```

Comment:
Why this group ?

Reply:
I'm going to change the groups to the basic sales permission, but I didn't remember the xml_id at the moment so I just placeholded it so once I could replace them all in one go